packages/gradle/project-graph/src/main/kotlin/dev/nx/gradle/utils/CiTargetsUtils.kt

```diff
@@ -88,8 +88,13 @@ private fun buildTestCiTarget(
     testFile: File,
     testTask: Task,
     projectRoot: String,
-    workspaceRoot: String
+    workspaceRoot: String,
 ): MutableMap<String, Any?> {
+  val taskInputs = getInputsForTask(testTask, projectRoot, workspaceRoot, null)
+  val testFileInput = replaceRootInPath(testFile.path, projectRoot, workspaceRoot)
```

Comment:
Is the test file not part of the other task inputs? Would this repeat it twice?

Reply:
change to taskInputs